libs/template/README.md

```diff
@@ -82,7 +84,23 @@ import { UnpatchModule } from '@rx-angular/template/unpatch';
   declarations: [...],
   imports: [ForModule, LetModule, PushModule, UnpatchModule],
 })
-export class MyModule {}
+export class AnyModule {}
+```
+
+Standalone component setup:
+
+```typescript
+import { LetModule } from '@rx-angular/template/let';
+import { ForModule } from '@rx-angular/template/for';
+import { PushModule } from '@rx-angular/template/push';
+import { UnpatchModule } from '@rx-angular/template/unpatch';
+
+@Component({
+  standalone: true,
+  imports: [ForModule, LetModule, PushModule, UnpatchModule],
+  template: `...`,
+})
+export class AnyComponent {}
```

Comment:
I would suggest to stick to the standalone approach only, no need to keep the module-based approach IMO.

Note that we removed all the modules, `ForModule` should be replaced with `RxFor`, `LetModule` with `LetDirective`, `PushModule` by `PushPipe`, and `UnpatchModule` with `UnpatchDirective`.